src/Microsoft.Health.Fhir.Core/Features/Conformance/CapabilityStatementBuilder.cs

```diff
@@ -184,13 +177,12 @@ public ICapabilityStatementBuilder AddGlobalSearchParameters()
             return this;
         }
 
-        private async Task<ICapabilityStatementBuilder> SyncSearchParamsAsync(string resourceType, CancellationToken cancellationToken)
+        private ICapabilityStatementBuilder SyncSearchParamsAsync(string resourceType)
```

Comment:
Low priority: we should rename this method. I had the impression it was async.

Reply:
I missed that. Without seach parameter status mamager they went back to synch calls.